### ISSUE TITLE
Not work with 1.15.1

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#37]
```
[00:26:47] [Server thread/INFO]: [EnderContainers] Enabling EnderContainers v2.2.0
[00:26:47] [Server thread/ERROR]: [EnderContainers] 
[00:26:47] [Server thread/ERROR]: [EnderContainers] >> --------------- <<
[00:26:47] [Server thread/ERROR]: [EnderContainers] >> CRITICAL ERROR! <<
[00:26:47] [Server thread/ERROR]: [EnderContainers] >> --------------- <<
[00:26:47] [Server thread/ERROR]: [EnderContainers] 
[00:26:47] [Server thread/ERROR]: [EnderContainers] >> Error when loading Yaml for class Locale! Yaml value COMMANDS.BACKUPS.CREATED cannot be parsed!
[00:26:47] [Server thread/ERROR]: [EnderContainers] >> Are you sure that value '&aBackup created with the name &b%backup%&a.' is good?
[00:26:47] [Server thread/ERROR]: [EnderContainers] 
[00:26:47] [Server thread/ERROR]: [EnderContainers] Cannot parse the yaml file
java.lang.NullPointerException: null
	at fr.utarwyn.endercontainers.compatibility.ServerVersion.isOlderThan(ServerVersion.java:51) ~[?:?]
	at fr.utarwyn.endercontainers.configuration.Locale.parseValue(Locale.java:164) ~[?:?]
	at fr.utarwyn.endercontainers.configuration.YamlFile.load(YamlFile.java:60) ~[?:?]
	at fr.utarwyn.endercontainers.configuration.Files.initLocale(Files.java:72) ~[?:?]
	at fr.utarwyn.endercontainers.EnderContainers.onEnable(EnderContainers.java:58) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.15.1.jar:git-Paper-29]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.15.1.jar:git-Paper-29]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.15.1.jar:git-Paper-29]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:468) ~[patched_1.15.1.jar:git-Paper-29]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:382) ~[patched_1.15.1.jar:git-Paper-29]
	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:481) ~[patched_1.15.1.jar:git-Paper-29]
	at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.15.1.jar:git-Paper-29]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:884) ~[patched_1.15.1.jar:git-Paper-29]
	at java.lang.Thread.run(Unknown Source) [?:?]
[00:26:47] [Server thread/INFO]: [EnderContainers] Disabling EnderContainers v2.2.0
[00:26:47] [Server thread/ERROR]: [EnderContainers] Cannot load the plugin's locale. Please check the above log. Plugin loading failed.
```

**Comment by andris155** [comment]
https://github.com/utarwyn/EnderContainers/pull/38

**Comment by utarwyn** [comment]
Thank you for the PR, it's now merged. I have to work on some adjustements to be sure that all features of the plugin are working with 1.15 versions.

**Comment by utarwyn** [comment]
I have tested all features of the plugin on 1.15 versions, **and now it seems to work well**.

There **was a problem with holograms** because Mojang has changed the protocol to spawn entities (more info [on this Spigot forum post](https://www.spigotmc.org/threads/holograms-break-on-1-15.408675/)). But I have fix this thanks to the commit 3de717efa4c38977a6e0fc21e2a7f0d3341fb0ba.
